src/analyzer/protocol/smtp/SMTP.h

```diff
@@ -90,6 +91,7 @@ class SMTP_Analyzer final : public analyzer::tcp::TCP_ApplicationAnalyzer {
     std::unique_ptr<detail::SMTP_BDAT_Analyzer> bdat; // if set, BDAT chunk transfer active
 
     analyzer::mime::MIME_Mail* mail;
+    std::string mail_fuid; // fuid for mail data file analysis
```

Comment:
More naming: `mail` is already used differently in the line just above, so how about naming this, say, `rfc822_fuid`?